core/src/main/java/org/mule/runtime/core/internal/management/stats/DefaultFlowsSummaryStatistics.java

```diff
@@ -66,6 +74,7 @@ public class DefaultFlowsSummaryStatistics implements FlowsSummaryStatistics {
   private final AtomicInteger activeTriggerFlows = new AtomicInteger(0);
   private final AtomicInteger declaredApikitFlows = new AtomicInteger(0);
   private final AtomicInteger activeApikitFlows = new AtomicInteger(0);
+  private final Map<FlowType, Set<String>> flowTypeMap = new ConcurrentHashMap<>();
```

Comment:
since flowtype is only 3 possible values, it would be better to have 3 sets rather than a map